OneCommunityGlobal/HighestGoodNetworkApp
# Zuhang fix team management alert if user already exists

# Description
(PRIORITY MEDIUM) Jae: Fix Team Management alert if user already exists
a. Other Links -> Teams -> Choose Team → Members → Add Existing
b. If you try to add an existing member, it says “Please choose a valid user.” It should say, “Member is already a part of this team.”

## Main changes explained:
- Address a bug in file src/components/Teams/MembersAutoComplete.jsx.
- Update file src/components/Teams/TeamMembersPopup.jsx. Adding a new state called "duplicateUserAlert" to detect duplicate member.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. go to dashboard→ Other Links→ Teams→Members
5. Add new members or existing members

## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/123051008/4cb27aa9-2b9d-48d5-82af-8c5e3047c396
